### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -12707,6 +12710,9 @@ flage->flags
 flaged->flagged
 flages->flags
 flagg->flag
+flahs->flash
+flahsed->flashed
+flahses->flashes
```
Comment (sebweb3r) [inline codespell_lib/data/dictionary.txt:12715]:
```suggestion
flahses->flashes
flahsing->flashing
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -13141,6 +13147,7 @@ frequences->frequencies
 frequencey->frequency
 frequenct->frequency
 frequentily->frequently
+frequeny->frequency, frequently,
```
Comment (sebweb3r) [inline codespell_lib/data/dictionary.txt:13150]: frequenies
```suggestion
frequeny->frequency, frequently,
frequenies->frequencies
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -13141,6 +13147,7 @@ frequences->frequencies
 frequencey->frequency
 frequenct->frequency
```
Comment (sebweb3r) [inline codespell_lib/data/dictionary.txt:13148]:
```suggestion
frequenct->frequency
frequenies->frequencies
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -5921,6 +5923,7 @@ compromize->compromise
 compsable->composable
 compsite->composite
 comptabile->compatible
+comptability->compatibility
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:5926]: Perhaps this should also suggest `computability` ?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -12707,6 +12710,10 @@ flage->flags
 flaged->flagged
 flages->flags
 flagg->flag
+flahs->flash
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:12713]: Also suggest `flags` ?

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -13140,7 +13147,9 @@ frequantly->frequently
 frequences->frequencies
 frequencey->frequency
 frequenct->frequency
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:13149]: It not one that you added, but perhaps this should be `frequenct->frequency, frequent,`

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -13140,7 +13147,9 @@ frequantly->frequently
...
+frequenies->frequencies
 frequentily->frequently
+frequeny->frequency, frequently,
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:13152]: This should also suggest `frequent`

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -19856,6 +19872,7 @@ pasttime->pastime
 pastural->pastoral
 pasword->password
 paswords->passwords
+patchs->patches
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:19875]: Also suggest `paths` ?

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -27797,6 +27817,7 @@ trianglular->triangular
 trianglutaion->triangulation
 triangulataion->triangulation
 triangultaion->triangulation
+triger->trigger
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:27820]: Also suggest `tiger`? :tiger:

**Comment by luzpaz** [comment]
Added requested revisions in 3672175

**Pull request merged**